Modules/Devices/src/Devices.Application/Identities/Commands/ApproveDeletionProcess/Handler.cs

```diff
@@ -47,7 +47,7 @@ public async Task<ApproveDeletionProcessResponse> Handle(ApproveDeletionProcessC
 
         _eventBus.Publish(new TierOfIdentityChangedDomainEvent(identity, oldTierId, newTierId));
 
-        var daysUntilDeletion = deletionProcess.GracePeriodEndsAt?.DaysUntilDate() ?? throw new Exception("Expected GracePeriodEnd to be set but found null instead.");
+        var daysUntilDeletion = deletionProcess.GracePeriodEndsAt?.DaysUntilDate() ?? throw new Exception($"Expected '{deletionProcess.GracePeriodEndsAt}' to be set but found 'null' instead.");
```

Comment:
Didn't you notice that my suggestion didn't make sense? ;)

```suggestion
        var daysUntilDeletion = deletionProcess.GracePeriodEndsAt?.DaysUntilDate() ?? throw new Exception($"Expected '{nameof(deletionProcess.GracePeriodEndsAt)}' to be set but found 'null' instead.");
```

Reply:
Minor detail, I did not notice it, no 😅